server/grpc_service.go

```diff
@@ -633,14 +635,16 @@ func (s *tsoServer) Recv(timeout time.Duration) (*pdpb.TsoRequest, error) {
 		request, err := s.stream.Recv()
 		requestCh <- &pdpbTSORequest{request: request, err: err}
 	}()
+	timer := time.NewTimer(timeout)
```

Comment:
How to choose between a ticker and a timer? I find in this PR, some places use tickers, and others use timers.

Reply:
IMO, if need to retry some times, ticker is more suitable, if only to use once, timer is more suitable.

Reply:
Yes, but seem we mixed them.

Reply:
Which ones do you think need to be modified?

Reply:
like PatrolRegions maybe?

Reply:
`PatrolRegions` and `runScheduler` use timer and timer.Reset, I will replace them with ticker

Reply:
oh, I noticed that we need some intervals to be configurable.

Reply:
revert it and add comments